### ISSUE TITLE
`comparisons_to_deactivate` should take strings of `output_column_name` not Comparison

### ISSUE THREAD

**Issue by RobinL** [moj-analytical-services/splink#526]
Otherwise user has to do:

```
n = linker._settings_obj._get_comparison_by_output_column_name("name")
linker.estimate_parameters_using_expectation_maximisation(
    tb1, comparisons_to_deactivate=[n]
)
```

**Comment by RobinL** [comment]
Warning should be given if `comparison_levels_to_reverse_blocking_rule` note also given

**Comment by RobinL** [comment]
Closed by #543
